src/find_svn_rev.py

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#   Copyright 2016 WebAssembly Community Group participants
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import re
+import subprocess
+import sys
+
+
+COMMIT_MATCH = re.compile('^commit ([0-9a-f]+)')
+REV_MATCH = re.compile('git[-]svn[-]id[:].*[@]([0-9]+)')
+
+
+def FindGitCommit(commit):
```

Comment:
This function is confusingly named because it doesn't find/search for anything, and it returns an SVN rev not a git commit :)
How about `GetSvnRev` or `SvnRevFromGitCommit` or something like that?